### ISSUE TITLE
A question about the testing result

### ISSUE THREAD

**Issue by tyj117** [roomylee/cnn-relation-extraction#6]
Hi,
I tried to train the model by using SemEval2010_task8 data set, but the result is not as good as the paper 'Perspective from Convolutional Neural Networks', the F1-measure of my result is only 74%.
Can you tell me your result?
Thanks
Niko

**Comment by roomylee** [comment]
@tyj117 
Hi, Niko. I'm sorry for late.
I change a optimizer from Adam to AdeDelta and fix some hyperparameters according to the paper.
The project was updated with a new version that reproduces the performance from the paper.
I got F1-score of 82.27%  on the official metric. Check it out. https://github.com/roomylee/cnn-relation-extraction#results
